lib/OpenQA/Schema.pm

```diff
@@ -36,41 +36,35 @@ our $VERSION = 75;
 
 __PACKAGE__->load_namespaces;
 
-sub _get_schema {
-    state $schema;
-    return \$schema;
-}
-
+my $SINGLETON;
```

Comment:
Is it a common pattern to name a variable `$SINGLETON` in caps? Because it looks a bit odd and the name is not very intuitive (which singleton instance is it?).

Reply:
Yes, it's convention to uppercase package scope variables.

Reply:
When i use the term singleton it's always the singleton instance of the current class.